dd-sdk-android/src/main/kotlin/com/datadog/android/rum/GlobalRum.kt

```diff
@@ -29,6 +29,7 @@ import java.util.concurrent.atomic.AtomicReference
  */
 object GlobalRum {
 
+    internal val globalAttributes: MutableMap<String, Any?> = mutableMapOf()
```

Comment:
hmm I think you should use a ConcurrentHashMap here and also add some unit tests for multithread safety.